.travis.yml

```diff
@@ -5,10 +5,6 @@ node_js:
 sudo: required
 dist: trusty
 
-branches:
-  only:
-    - master
```

Comment:
Yes, I wanted to run the test-cases on travis. They pass on my machine. I will revert it back.